docs/gettsim_objects/input_variables.md

```diff
@@ -572,3 +574,23 @@ Type: int
 Number of own children below the age of 25 (incl. not in hh)
 
 Type: int
+
+## `erwerbsgemindert`
```

Comment:
```suggestion
## `voll_erwerbsgemindert`
```
I think that will make things clearer, but feel free to leave it as is if I am misunderstanding something.